gpii/node_modules/ontologyHandler/test/node/OntologyHandlerTests.js

```diff
@@ -79,6 +79,159 @@ https://github.com/gpii/universal/LICENSE.txt
         ]
     };
 
+    var floydWarshallTestsLength = 18;
+    var floydWarshallTests = [
+        {
+            name: "Initial 4 vertice, 3 edges",
```

Comment:
I am craZ for sure.  But what I meant, is that on the above comment you mentioned to replace it to vertex, but I believe in all the instances in OntologyHandlerTests it's actually plural, so it could either go to vertices or vertexes, but since the replacement you suggested above was to vertex, it seemed like the preferred version of the plural as well.